build.xml

```diff
@@ -371,6 +384,7 @@
         <mkdir dir="${build.dir.lib}"/>
         <mkdir dir="${jacoco.export.dir}"/>
         <mkdir dir="${jacoco.partials.dir}"/>
+        <exec executable="${basedir}/dev-support/install-git-hooks.sh" osfamily="unix" dir="${basedir}" logError="true" failonerror="true" failifexecutionfails="true" />
```

Comment:
this noop if the hooks are already installed, they check for existence and exit early